src/Oci8/Schema/Grammars/OracleGrammar.php

```diff
@@ -168,7 +168,7 @@ public function compileTableExists()
      */
     public function compileColumnExists($table)
     {
-        return "select column_name from user_tab_columns where table_name = upper('" . $table . "')";
+        return "select column_name from all_tab_cols where table_name = upper('" . $table . "')";
```

Comment:
I think same case as `all_tables` would apply here where we need to specify the owner.